executing/_position_node_finder.py

```diff
@@ -703,6 +735,57 @@ def node_match(node_type: Union[Type, Tuple[Type, ...]], **kwargs: Any) -> bool:
             if node_match(ast.FormattedValue) and inst_match("FORMAT_VALUE"):
                 return
 
+        if sys.version_info >= (3, 13):
+            if (
+                node_match(ast.FunctionDef)
```

Comment:
What about classes, lambdas, and async functions?

Reply:
It looks like this is not needed any more. I will check it.